osu.Game/Screens/Select/Leaderboards/BeatmapLeaderboard.cs

```diff
@@ -182,9 +194,22 @@ protected override APIRequest FetchScores(Action<IEnumerable<ScoreInfo>> scoresC
             return req;
         }
 
-        protected override LeaderboardScore CreateDrawableScore(ScoreInfo model, int index) => new LeaderboardScore(model, index, IsOnlineScope)
+        protected override LeaderboardScore CreateDrawableScore(ScoreInfo model, int index)
         {
-            Action = () => ScoreSelected?.Invoke(model)
-        };
+            model.Beatmap = beatmap;
```

Comment:
What's this about?

Reply:
I have found that the `ScoreInfo` actually does not contain the `BeatmapInfo` for some reason. So this is to make sure that the score have that information.

Reply:
Feels like this should be fixed somewhere else, probably where it's created.